src/FAudio_internal.c

```diff
@@ -525,7 +525,7 @@ static void FAudio_INTERNAL_MixSource(FAudioSourceVoice *voice)
 	}
 
 	/* Filters */
-	if (voice->flags & FAUDIO_VOICE_USEFILTER)
+	if (voice->flags & FAUDIO_VOICE_USEFILTER && voice->filterState != NULL)
```

Comment:
This can be removed, seems smarter to just allocate on `USEFILTER` right away:

https://github.com/flibitijibibo/FACT/commit/f6ef20b5294f9451695ec8090b8aaf3000104f8b